core/src/code.rs

```diff
@@ -202,6 +208,44 @@ pub struct Code {
     instruction_weights_version: u32,
 }
 
+fn check_for_mut_global_exports(module: &Module) -> Result<(), CodeError> {
+    let global_exports_indexes = module
```

Comment:
please no, don't put it in the protocol 

just make it const in instrumentantion or whatever

Reply:
Do you mean make __gear_stack_end export to const global?

Reply:
Done